AEPMessaging/Sources/Messaging+State.swift

```diff
@@ -20,7 +20,7 @@ extension Messaging {
         guard let cachedPropositions = cache.propositions else {
             return
         }
-        propositions = cachedPropositions
+        cbePropositions = cachedPropositions
```

Comment:
The `cachedPropositions` here are the IAM messages stored in the file using cache service. We never write cbe propositions to the file. Can we rename it?

Reply:
But the ones on line 39 are cbe propositions 😅 we can rename it to generic `cachedPropositions` or `cbeAndIAMPropositions`

Reply:
i went with `inMemoryPropositions`

i wonder if there's a better way to consolidate all our propositions into one container - or at least make them separated in a more meaningful way.  i'll explore that if we have time before the release.